packit_service/worker/handlers/bodhi.py

```diff
@@ -133,12 +135,22 @@ def run(self) -> TaskResults:
             issue_repo = self.service_config.get_project(
                 url=self.job_config.issue_repository
             )
-            body = (
-                f"Bodhi update failed to be created for `{self.koji_build_event.nvr}`:\n"
-                "```\n"
-                f"{ex}\n"
-                "```"
-            )
+
+            if isinstance(ex.__cause__, AuthError):
+                body = (
+                    f"Bodhi update failed to be created for `{self.koji_build_event.nvr}` "
```

Comment:
```suggestion
                    f"Bodhi update creation failed for `{self.koji_build_event.nvr}` "
```

looks like a simpler wording to me